packages/jest-expo/src/preset/setup.js

```diff
@@ -162,6 +162,8 @@ jest.doMock('react-native/Libraries/LogBox/LogBox', () => ({
 try {
   jest.mock('expo-modules-core', () => {
     const ExpoModulesCore = jest.requireActual('expo-modules-core');
+    const uuid = jest.requireActual('expo-modules-core/src/uuid/uuid.web');
```

Comment:
Does it even work? This requires a TS file in a JS context 🤔 We should rather use the compiled version from `build` folder.

```suggestion
    const uuid = jest.requireActual('expo-modules-core/build/uuid/uuid.web');
```

Reply:
Well it did work, but you are right, importing from the build folder makes sense 😅